Client/Frontend/Fakespot/Views/FakespotHighlightsCardView.swift

```diff
@@ -196,7 +196,7 @@ class FakespotHighlightsCardView: UIView, ThemeApplicable {
 
     private func recordTelemetry() {
         TelemetryWrapper.recordEvent(category: .action,
```

Comment:
Telemetry looks like something that belongs to business logic and should be out of the view. I imagine e better place would be on the action on view model?